Misc/NEWS.d/next/Core and Builtins/2022-03-08-21-59-57.bpo-46962.UomDfz.rst

```diff
@@ -0,0 +1,10 @@
+Classes and functions that unconditionally declared their docstrings
+ignoring --without-doc-strings compilation flag no longer do so.
+
+The classes affected are :class:`~ctypes.UnionType`,
+:class:`~pickle.PickleBuffer`, :class:`~testcapi.RecursingInfinitelyError`,
+and :class:`~types.GenericAlias`.
```

Comment:
I don’t think the module names should be hidden here.

Reply:
Fixed.